### DISCUSSION TITLE
Added: C# Language Design Notes for May 16, 2017

### DISCUSSION THREAD

**Discussion by MadsTorgersen** [dotnet/csharplang#657] [General]
[C# Language Design Notes for May 16, 2017](https://github.com/dotnet/csharplang/blob/master/meetings/2017/LDM-2017-05-16.md)

1. Triage C# 7.1 features that didn't make it
2. Look at C# 7.2 features
3. GitHub procedure around new design notes and proposals
4. Triage of championed features

Please discuss.

**Comment by HaloFour** [comment]
Re "CallerArgumentExpression", did the LDM weigh in on any of the additional suggested attributes to get more information about the caller's type, namespace, method, etc.?  #87

**Comment by miloush** [comment]
Hmm... I was disappointed the leading digit separators did not work recently, if only I remembered... I think I wanted to align some numbers and the leading zeroes were being removed.

**Comment by tannergooding** [comment]
👍 for static delegates.
